src/cljc/quil/core.cljc

```diff
@@ -4356,6 +4356,12 @@
   [size]
   (.textSize (current-graphics) (float size)))
 
+(defn get-text-size
+  "Return the current text size, as a float. Font size is measured in units
```

Comment:
Please add metadata to the function like in all other functions. If there is no corresponding processing function - put nil ([example](https://github.com/quil/quil/blob/master/src/cljc/quil/core.cljc#L3204).